### PR TITLE
Fix: Enable text wrapping in docstring header on smaller screens (#2293)

### PR DESCRIPTION
**Pull request by QuantuM410**
### Changes:
overflow: auto CSS property added to the header tag.

[screen-recorder-tue-oct-24-2023-13-13-39.webm](https://github.com/JuliaDocs/Documenter.jl/assets/85943021/0a3d734a-9b8c-4d84-9e58-611f43c6b85c)

### CODE REVIEW

**Review by mortenpi — COMMENTED** [review]
Thanks! This basically LGTM, but could you:

1. Undo the unrelated formatting changes? I think the original formatting is preferred and more consistent with the rest of the code, and also I would prefer not to mix unrelated changes in PRs.
2. Add a quick note about this into `CHANGELOG.md`.

**Comment by QuantuM410** [comment]
> Thanks! This basically LGTM, but could you:
> 
>     1. Undo the unrelated formatting changes? I think the original formatting is preferred and more consistent with the rest of the code, and also I would prefer not to mix unrelated changes in PRs.
> 
>     2. Add a quick note about this into `CHANGELOG.md`.

While I am adding the note `CHANGELOG.md`, do I also have to run `make changelog`. Once I am running make changelog two lines are being added (one unrelated to the issue but is related to the previous fix in `v1.1.2`)
image

image

**Comment by mortenpi** [comment]
Yeah, if you don't mind, run `make changelog`. And let's add the 2300 too here -- that was an oversight in the earlier PR.

**Comment by QuantuM410** [comment]
@mortenpi I have updated the files with required changes

File: assets/html/scss/documenter/components/_docstring.scss [hunk 1]
```diff
@@ -71,4 +72,4 @@
   & > section:hover a.docs-sourcelink {
     opacity: 1;
   }
-}
+}
```
Comment (mortenpi) [inline assets/html/scss/documenter/components/_docstring.scss:75]:
```suggestion
}

```

**Review by mortenpi — APPROVED** [review]
Thank you!

**Pull request merged**